### PR TITLE
Fixes #75 "Use email other than settings.DEFAULT_FROM_EMAIL".

### LINKED ISSUE #75: Use email other than settings.DEFAULT_FROM_EMAIL
Hi,

I don't want to use settings.DEFAULT_FROM_EMAIL as the email address that emails are sent from. I propose adding a settings option called REGISTRATION_FROM_EMAIL and if it exists RegistrationProfile._send_mail() & send_notification_email_reciver() will use settings.REGISTRATION_FROM_EMAIL  and if it doesn't exist they will use settings.DEFAULT_FROM_EMAIL.

I will make the code changes and test and will do a pull request later today.

Thanks
--Mick

### PR DESCRIPTION
**Pull request by mick-t**
See https://github.com/lambdalisue/django-inspectional-registration/issues/75

### CODE REVIEW

File: src/registration/contrib/notification/__init__.py [hunk 1]
```diff
@@ -142,6 +143,15 @@ def send_notification_email_reciver(sender, user, profile, request, **kwargs):
...
     # remove duplications
     recipients = frozenset(recipients)
+    
+    try:
+        mail_from = settings.REGISTRATION_FROM_EMAIL
+    except AttributeError as e:
+        logger.warning('%s:%s:settings.REGISTRATION_FROM_EMAIL not'
+                       ' found: %s' % (
+                            __file__, sys._getframe().f_code.co_name, e)
+                       )
```
Comment (lambdalisue) [inline src/registration/contrib/notification/__init__.py]: It should not be warning.

**Comment by lambdalisue** [comment]
Please fix the code to pass CI and what I pointed. I'm ok to add a new variable but I don't like to make the previous way deprecated.

File: src/registration/contrib/notification/__init__.py (hunk 1 above)
Comment (mick-t) [inline src/registration/contrib/notification/__init__.py]: What would you like it to be?

Comment (giginet) [inline src/registration/contrib/notification/__init__.py]: I think you should raise `ImproperlyConfigured` instead.

File: src/registration/contrib/notification/__init__.py [hunk 2]
```diff
@@ -142,6 +143,15 @@ def send_notification_email_reciver(sender, user, profile, request, **kwargs):
                     ) % method_or_iterable)
     # remove duplications
     recipients = frozenset(recipients)
+    
+    try:
```
Comment (mick-t) [inline src/registration/contrib/notification/__init__.py]: I agree, that works perfectly and removes the logger.

**Comment by mick-t** [comment]
@lambdalisue I updated the code with your suggestions. Thanks for making me aware of get()! 👍 

I agree that the previous method should not be deprecated.

**Comment by lambdalisue** [comment]
Oops.  I'm really sorry I'm too Vimmer. The correct one should be `settings.get` and `get` was a Vim's function...

**Comment by mick-t** [comment]
get() works. I tested it with and without the REGISTRATION_FROM_EMAIL in the settings file.

**Comment by lambdalisue** [comment]
https://travis-ci.org/lambdalisue/django-inspectional-registration/jobs/174312048#L225

**Comment by mick-t** [comment]
Looks like nothing breaks this time! :)

**Comment by lambdalisue** [comment]
LGTM Thanks!

**Comment by lambdalisue** [comment]
I'll upload a new version which include this commit tonight. Ping me if I forget about this

**Pull request merged**